middle_end/flambda2/to_cmm/to_cmm_expr.ml

```diff
@@ -281,7 +282,7 @@ let translate_raise env res apply exn_handler args =
           "Apply_cont calls an exception handler without a Pop trap action:@ %a"
           Apply_cont.print apply
     in
-    let dbg = Apply_cont.debuginfo apply in
+    let dbg = Env.add_inlined_debuginfo env (Apply_cont.debuginfo apply) in
```

Comment:
Here and below there are various cases all of which come from Apply_cont expressions (presumably via the apply_cont function below?).  If so, it would be clearer to put this add_inlined_debuginfo call right at the top of the apply_cont function, instead of in these various other places.

Reply:
done, these function now take a `~dbg_with_inlined` parameter